README.md

```diff
@@ -25,3 +25,9 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+"I certify that I have listed all sources used to complete this exercise, including the use of any Large Language Models. All of the work is my own, except where stated otherwise. I am aware that plagiarism carries severe penalties and that if plagiarism is suspected, charges may be filed against me without prior notice."
+
+I had the help of chat gpt to help me figure out how to split into 3 sub arrays instead of 2 subarrays aswell as reminding me how to get the origonial time complexity formula.
+
+Each subaray is n/3 so the recurrence relation for T(n) should be T(n) = 3T(n/3) + O(1). The 3 is because there are three arrays and the T for the recrusion. The big O is a constant of 1 because of the singular act of adding the elements. After we get the equation we can compute n (log of b) a. We can sub in b for the times we are using the recursion which is 3 and we can sub in for a with 3 being the parts we split n into. We get log of 3 to 3 giving us n^1. Next we need to take that in context of the big O function of O(1). Since we know that O(1) has a smaller rate of change than n^logb of a we can have T(n) = Theta(n) be the complexity.
```

Comment:
Please use the correct notation to denote that T(n) is an element of a particular complexity class.

Reply:
is the updated notation better?